winui/AvatarView/Overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Overview of WinUI AvatarView control | Syncfusion
+description: Learn all about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more here.
+platform: WinUI
+control: SfAvatarView
+documentation: ug
+---
+
+# WinUI Avatar View (SfAvatarView) Overview
+
+The AvatarView control provides a graphical representation of user image that allows you to customize the view by adding image, background color, icon, text, etc.
+
+![AvatarView control overview in WinUI](AvatarView_images/winui_avatarview_overview.png)
```

Comment:
@MadhumithanJayapathi WinUI AvatarView control overview. use small letters for folder name of images.

Reply:
@Manivannan-E Used small letters for folder name for images